.github/workflows/build.yml

```diff
@@ -36,24 +37,23 @@ jobs:
       - name: Run the tests
         run: npm test
 
-      - name: Check for version change
-        id: version-check
-        run: |
-          [ "$(git show head^:package.json | jq .version)" = "$(cat package.json | jq .version)" ] && echo "::set-output name=version_changed::true" || echo "::set-output name=version_changed::false"
-
       - name: Copy relevant documentation into the build folder
-        if: steps.version-check.outputs.version_changed == 'true'
         run: cp package.json README.md LICENSE-APACHE LICENSE-MIT ./build
 
       - name: Publish to npmjs
         uses: JS-DevTools/npm-publish@v1
-        if: github.ref == 'refs/heads/master' && steps.version-check.outputs.version_changed == 'true'
+        if: github.ref == 'refs/heads/master'
         with:
           token: ${{ secrets.NPM_TOKEN }}
           package: build/package.json
           # don't try and deploy if versions haven't changed
           check-version: true
```

Comment:
Won't that publish to npmjs all new commits, not just new releases?

Reply:
nah, the `check-version` below verifies the version. My git magic has some holes in it if we have master commits that don't change the version, but it's only for tagging and releases so it isn't a big deal